qt/mainwindow.cpp

```diff
@@ -370,14 +370,12 @@ void MainWindow::OnSearchButtonClicked()
   }
 }
 
-void MainWindow::OnEngineCreated()
+void MainWindow::OnEngineCreating()
```

Comment:
The name is confusing now. If it is a callback, is it called before creating engine or after it? If it is not a callback, why it starts with On... prefix?